pyproject.toml

```diff
@@ -35,7 +36,7 @@ poetry = "^1.7.1"
 poetry-plugin-export = "^1.6.0"
 functions-framework = "^3.5.0"
 cron-validator = "^1.0.8"
-prediction-market-agent-tooling = { version = "^0.40.0", extras = ["langchain", "google"] }
+prediction-market-agent-tooling = { version = "^0.40.0", extras =  ["google"] }
```

Comment:
Hmm I think this could be a problem, if PMAT and PMA use different versions of langchain, as the langchain features from PMAT are used in this repo.

In this case, do we actually need `langchain-chroma` for the test? We already have the `chromadb` package. If we do need it, then I think it should be updated in PMAT first.

Reply:
Chroma is not the problem - problem is, Langchain >=0.2.6 is required for the embeddings to work.
The way I got around it is by bypassing the langchain requirement from PMAT - is updating langchain on PMAT the suggested approach?

Reply:
> is updating langchain on PMAT the suggested approach?

In that case, yes

> Langchain >=0.2.6 is required for the embeddings to work.

What library needs this? `langchain_openai`?

Reply:
Yes, `langchain_openai` because of `OpenAIEmbeddings`